variables.tf

```diff
@@ -41,6 +59,13 @@ variable "cos_instance_crn" {
   description = "CRN of the Cloud Object Storage to store SCC data"
 }
 
+
+variable "wp_instance_crn" {
+  type        = string
+  description = "CRN of the Workload Protection instance to attach to SCC instance"
```

Comment:
Optionally pass the CRN of an existing SCC Workload Protection instance to attach it to the SCC instance.